src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -53,7 +60,13 @@ const ExpandedNode: FC<ExpandedNodeProps> = ({
 
     // Create DocumentReference and add to patient record(and post to FHIR server)
     if (note) {
-      const noteString = createNoteContent(note, patientRecords, status, comments, pathwayState);
+      const noteString = createNoteContent(
+        note,
+        patientRecords,
+        status,
+        comments ?? '',
```

Comment:
Is this needed? It seems like we can just grab the comments from `note.notes`

Reply:
Can't remember why I did this, but it doesn't seem necessary.